R/agg_scale.R

```diff
@@ -525,9 +489,22 @@ assert_agg_scale_args <- function(dt,
                                   col_type,
                                   mapping,
                                   agg_function,
+                                  missing_dt_severity,
+                                  overlapping_dt_severity,
                                   collapse_interval_cols,
                                   functionality) {
 
+  # check `missing_dt_severity` argument
+  assertthat::assert_that(
+    assertthat::is.string(missing_dt_severity),
+    missing_dt_severity %in% c("skip", "stop", "warning", "message", "none")
```

Comment:
Small thing, but maybe create a list object with these options, then reference that list? Instead of repeating it manually.